core/testsuite/src/test/java/com/blazebit/persistence/testsuite/WindowFunctionTest.java

```diff
@@ -579,6 +575,14 @@ public void testParsedNamedWindowDefinitionWithRange() {
                 .orderByAsc("per.age")
                 ;
 
+        String queryString = criteria.getQueryString();
+
+        assertEquals("SELECT per.age, window_sum(per.age,'ORDER BY',per.age,'ASC NULLS LAST', " +
+                        "'ROWS','BETWEEN','UNBOUNDED PRECEDING','AND','CURRENT ROW') " +
+                        "FROM Person per WINDOW x AS (ORDER BY per.age ASC  NULLS " +
```

Comment:
Same question here:
```suggestion
                        "FROM Person per WINDOW x AS (ORDER BY per.age ASC NULLS " +
```
?

Reply:
If it's easy to fix let's do it.